### PR TITLE
Proposal refactor

### PR DESCRIPTION
**Pull request by pratikasr**

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -169,6 +169,7 @@ import (
 	mv6 "github.com/comdex-official/comdex/app/upgrades/mainnet/v6"
 	mv7 "github.com/comdex-official/comdex/app/upgrades/mainnet/v7"
 	mv8 "github.com/comdex-official/comdex/app/upgrades/mainnet/v8"
+	tv8 "github.com/comdex-official/comdex/app/upgrades/testnet/v8_0_0"
```
Comment (dheerajkd30) [inline app/app.go]: change this to v9 upgrade @pratikasr

Comment (dheerajkd30) [reply app/app.go]: v9.beta is much better

File: app/upgrades/testnet/v8_0_0/upgrades.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+)
+
+func UpdateDenomAndAddAsset(ctx sdk.Context, assetKeeper assetkeeper.Keeper) {
+	asset, found := assetKeeper.GetAsset(ctx, 17)
+	if found {
+		asset.Denom = "ibc/50EF138042B553362774A8A9DE967F610E52CAEB3BA864881C9A1436DED98075"
+		assetKeeper.SetAsset(ctx, asset)
+	}
+
+	assetGDAI := assettypes.Asset{Name: "GDAI", Denom: "ibc/109DD45CF4093BEB472784A0C5B5F4643140900020B74B102B842A4BE2AE45DA", Decimals: sdk.NewInt(1000000000000000000), IsOnChain: false, IsOraclePriceRequired: true, IsCdpMintable: false}
```
Comment (dheerajkd30) [inline app/upgrades/testnet/v8_0_0/upgrades.go]: is it GDAI or grav.dai ? what's finalised ?

File: x/asset/client/cli/tx.go [hunk 3]
```diff
@@ -810,3 +810,127 @@ func NewCreateAssetInAppMsg(clientCtx client.Context, txf tx.Factory, fs *flag.F
 	}
 	return txf, msg, nil
 }
+
+func NewCmdSubmitAddMultipleAssetsPairsProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add-multiple-assets-pairs [flags]",
+		Args:  cobra.ExactArgs(0),
+		Short: "Submit multiple assets and pairs",
+		Long: `Must provide path to a add assets in JSON file (--add-assets) describing the asset in app to be created
```
Comment (dheerajkd30) [inline x/asset/client/cli/tx.go]: double check this, update the comment according to txs.

File: app/app.go (hunk 1 above)
Comment (pratikasr) [reply app/app.go]: Yes, Updated.

File: app/upgrades/testnet/v8_0_0/upgrades.go (hunk 2 above)
Comment (pratikasr) [reply app/upgrades/testnet/v8_0_0/upgrades.go]: GDAI is finalized.

**Review by dheerajkd30 — CHANGES_REQUESTED** [review]
changes requested

File: x/asset/client/cli/tx.go (hunk 3 above)
Comment (pratikasr) [reply x/asset/client/cli/tx.go]: Updated

**Review by dheerajkd30 — APPROVED** [review]

**Pull request merged**